config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
+  newTag: 0.97.1-7-gcafbc81a
```

Comment:
`make reset` to get rid of this.

Reply:
@iblancasa bump on this

Reply:
@iblancasa this is still here, we need to remove it prior to merge